common/djangoapps/student/tests/test_models.py

```diff
@@ -244,6 +255,173 @@ def test_enrollments_not_deleted(self):
         self.assertEqual(enrollment_refetched.all()[0], enrollment)
 
 
+@override_waffle_flag(REDIRECT_TO_COURSEWARE_MICROFRONTEND, active=True)
+@override_waffle_flag(COURSEWARE_MICROFRONTEND_PROGRESS_MILESTONES, active=True)
+@override_waffle_flag(COURSEWARE_MICROFRONTEND_PROGRESS_MILESTONES_STREAK_CELEBRATION, active=True)
+class UserCelebrationTests(SharedModuleStoreTestCase):
+    """
+    Tests for User Celebrations like the streak celebration
+    """
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.course = CourseFactory()
+        cls.course_key = cls.course.id  # pylint: disable=no-member
+        cls.STREAK_LENGTH_TO_CELEBRATE = UserCelebration.STREAK_LENGTHS_TO_CELEBRATE[0]
```

Comment:
Think the string replace might have gotten more than intended. Believe this (and others below) can just be lowercased.

Reply:
i had intentionally capitalized this constant for the tests also to make it easier to distinguish from the streak_length_to_celebrate return value being returned by perform_streak_updates in the tests